Data Explorer section enhancement

vitoc opened the pull request:
Use generic term for database and collection instead of example name that's not referenced anywhere else. Updated terms used for graphs and collection to reflect actual UI changes on portal. Added additional filter example and note.

azurecla commented:
Hi __@kevinscharpenberg__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (kevsch). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

vitoc committed:
Data Explorer section enhancement

Use generic term for database and collection instead of example name that's not referenced anywhere else. Updated terms used for graphs and collection to reflect actual UI changes on portal. Added additional filter example and note.

dennyglee reviewed (changes requested):
Please note that when using Data Explorer, the pane is called "Collections' per this [screenshot](https://github.com/dennyglee/notebooks/blob/master/images/graph-collections-pane.png?raw=true).  It may be confusing to readers when they do not see a pane that says "Graphs" in it with this change.
